data_collection/gazette/spiders/sp_guaratingueta.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import re
+
+import dateparser
+import scrapy
+from dateutil.rrule import YEARLY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpGuaratinguetaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3518404"
+    name = "sp_guaratingueta"
+    allowed_domains = ["https://guaratingueta.sp.gov.br/"]
```

Comment:
I think it is safe to remove this line, since `start_requests` is generating the ones you need. In case you want to keep it, you must adjust it to have only the domain. It will avoid this warning:

```
URLWarning: allowed_domains accepts only domains, not URLs. Ignoring URL entry https://guaratingueta.sp.gov.br/
```